package.json

```diff
@@ -5,13 +5,17 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "postinstall": "patch-package"
   },
   "dependencies": {
     "@emotion/react": "^11.1.5",
-    "next": "12.0.10",
+    "next": "^13.1.6",
```

Comment:
I guess you probably want the canary version (`13.1.7-canary.10`), so you can use `metadata` and `generateMetadata`

Reply:
https://github.com/upleveled/connecting-to-external-apis-secrets-with-stripe-fall-2022-vienna-austria/blob/85f6653d67eeb9ccaf099006071fdc8d8db4124b/package.json#L12